requirements.txt

```diff
@@ -1,12 +1,16 @@
 altair>=4.0.0,<5
 Click>=7.1.2
+colorama>=0.4.3
+cryptography>=3.2,<4.0.0
 importlib-metadata>=1.7.0 # (included in Python 3.8 by default.)
 ipywidgets>=7.5.1
 jinja2>=2.10
 jsonpatch>=1.22
 jsonschema>=2.5.1
 mistune>=0.8.4,<2.0.0
+nbformat>=5.0
 numpy>=1.14.1
+packaging
```

Comment:
should we pin this? or set a bound? just asking because we tend to do it for the others!

Reply:
Only reason I didn't was that every one of the dev dependencies that had some requirement that required packaging required "any"